src/LanguageServer.Engine/Handlers/ConfigurationHandler.cs

```diff
@@ -1,33 +1,20 @@
 using OmniSharp.Extensions.JsonRpc;
-using OmniSharp.Extensions.LanguageServer;
 using OmniSharp.Extensions.LanguageServer.Protocol;
 using OmniSharp.Extensions.LanguageServer.Protocol.Client.Capabilities;
-using OmniSharp.Extensions.LanguageServer.Protocol.Models;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
-using Newtonsoft.Json.Serialization;
 using Serilog;
-using Serilog.Events;
 using System;
 using System.Threading.Tasks;
-    
+
 namespace MSBuildProjectTools.LanguageServer.Handlers
 {
     using CustomProtocol;
-    using Handlers;
 
     /// <summary>
     ///     Language Server message handler that tracks configuration.
     /// </summary>
     public sealed class ConfigurationHandler
         : IDidChangeConfigurationSettingsHandler
     {
-        /// <summary>
-        ///     The JSON serialiser used to read settings from LSP notifications.
-        /// </summary>
-        /// <returns></returns>
-        readonly JsonSerializer _settingsSerializer = new JsonSerializer();
```

Comment:
Yes, I also removed a bunch of unused/unread fields. Lets not waste user's memory for them)

Reply:
Thanks - this was missed in a previous refactoring when the LSP stuff was still changing fairly frequently.